packages/cfd/src/features/Containers/cfd-password-manager-modal.tsx

```diff
@@ -194,20 +203,46 @@ const CFDPasswordManagerTabContent = ({
         setIsSubmitSuccessInvestor(true);
     };
 
+    React.useEffect(() => {
+        if (change_password_status === 'error' && change_password_error) {
+            showError((change_password_error as unknown as Error)?.message);
+        }
+        if (change_password_status === 'success') {
+            hideError();
+        }
+    }, [change_password_error, change_password_status]);
+
+    React.useEffect(() => {
+        if (change_investor_password_status === 'error' && change_investor_password_error) {
```

Comment:
can we move these value 'error', 'success' and 'investor' to constants instead?

Reply:
Sure